src/installer/mod.rs

```diff
@@ -137,7 +137,11 @@ impl Installer {
     ///
     /// If `config.old_root` is set, then home at that location will be retained.
     pub fn install(&mut self, mut disks: Disks, config: &Config) -> io::Result<()> {
-        let mut recovery_conf = RecoveryEnv::new()?;
+        let mut recovery_conf = if Path::new("/cdrom/recovery.conf").exists() {
+            None
+        } else {
+            Some(RecoveryEnv::new()?)
+        };
```

Comment:
Should this logic be reversed?
If path exists:
     recovery_conf = (path)
else:
     recovery_conf = None

Reply:
Yes, this caused it to try to parse the conf when it does not exist